src/ExportHistory/Models/ExportJobState.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask.ExportHistory;
+
+/// <summary>
+/// Export job state stored in the entity.
+/// </summary>
+public sealed class ExportJobState
```

Comment:
This feels very redundant to `ExportJobDescription`. Do we need both?

Reply:
yes, state is for whole metadata, description is for customer view.